[IMP] CRM: Pipeline Analysis page refresh (#17170)
odoo/documentation: content/applications/sales/crm/performance/win_loss.rst

Reviewer: After your rewrite - this doens't apply, I like this title now for the seciton =)

Pipeline Analysis
=================

The *CRM* app manages the sales pipeline as leads and opportunities move from stage to stage,
ultimately being either won or lost. After organizing the pipeline, the search options and reports
available on the *Pipeline Analysis* page can be used to gain insight into the effectiveness of the
pipeline and its users.

.. _win_loss/pipeline:

Pipeline analysis reports
=========================

To view the *Pipeline Analysis* page, go to :menuselection:`CRM app --> Reporting --> Pipeline`. A
stacked bar chart showcasing all leads created during the current year automatically loads. The bars
represent the number of leads currently in each stage of the sales pipeline, color-coded to show the
month the lead reached that stage.

.. image:: win_loss/pipeline-analysis-page.png

>    :alt: The default state of the Pipeline Analysis page is a graph, with many options to change it.

Reviewer: ```suggestion
   :alt: The default state of the Pipeline Analysis report is a graph, with many options to change it.
```
I think we can call it that, since we selected this specific report from the Reporting menu

- :guilabel:`Graph` view: Displays the data in a bar graph, allowing for quick visual comparisons of
  data across CRM stages. This is the default view.
- :guilabel:`Pivot` view: Displays the data in a customizable categorized metrics table, allowing
  for breakdowns of CRM data into specific categories or groups for more preceise analysis.
- :guilabel:`Cohort` view: Displays and organizes the data based on their :guilabel:`Created on` and
  :guilabel:`Closed Date` week, day, month, quarter, or year. This allows for easy identification of
  patterns over certain time periods. Week is the default setting.
- :guilabel:`List` view: Displays the data as a list of opportunities, allowing for easy viewing of
  detailed individual records.

.. _win_loss/search:

Filters and groupings
=====================

The :guilabel:`Pipeline Analysis` page can be customized with various filters and grouping options.

Filters
-------

The :guilabel:`Filters` section allows users to add pre-made and custom filters to the search
criteria. Multiple filters can be added to a single search.

- :guilabel:`My Pipeline`: Show leads assigned to the current user.
- :guilabel:`Active`: Show active leads.

> - :guilabel:`Inactive`: Show inactive leads.

Author: Clarified!

- :guilabel:`Conversion Date`: Groups the results by the date a lead was converted to an
  opportunity.
- :guilabel:`Expected Closing`: Groups the results by the date a lead is expected to close. Closed
  leads are marked as having been *Won*
- :guilabel:`Closed Date`: Groups the results by the date a lead was marked *Won*.
- :guilabel:`Lost Reason`: Groups the results by the reason selected when a lead was marked *Lost*.
- :guilabel:`Custom Group`: Allows the user to create a custom group with numerous options. See
  :ref:`Adding Custom Filters and Groups <win_loss/custom_filters>` for more information.

Favorites
---------

The :guilabel:`Favorites` section allows users to save regularly performed searches for later, so
that they don't need to be recreated every time. Multiple sets of search terms can be saved, shared
with others, or even set as the default for whenever the :guilabel:`Pipeline Analysis` page is
opened.

- :guilabel:`Save current search`: save the current search criteria for later.
- :guilabel:`Default filter`: when saving a search, check this box to make it the default search
  filter when the :guilabel:`Pipeline Analysis` page is opened.

> - :guilabel:`Shared`: when saving a search, check this box to make it available to other users.

Reviewer: ```suggestion

```
This is too basic and should not be in this doc. Reporting docs should be for the specific report only, and shouldn't include this kind of info.

  CRM pipeline as *Ongoing* or *Rotting*.
- :guilabel:`Inactive`: Show inactive opportunities. Inactive opportunities are those that appear in
  the CRM pipeline as *Lost*, *Won,* and *Archived*.
- :guilabel:`Won`: Show opportunities that have been marked *Won*.
- :guilabel:`Lost`: Show opportunities that have been marked *Lost*.
- :guilabel:`Created on`: Show opportunities that were created during a specific period of time. The
  default time period is the current year, but it can be adjusted as needed.
- :guilabel:`Expected Closing`: Show opportunities that are projected to close during a specific
  period of time. Closed opportunities are marked as having been *Won*.
- :guilabel:`Date Closed`: Show opportunities that have been marked *Won* during a specific period
  of time.
- :guilabel:`Archived`: Show opportunities that have been archived. *Lost* opportunities are
  automatically archived, but not all archived opportunities are marked as *Lost*.
- :guilabel:`Custom Filter`: Allows the user to create a custom filter with numerous options.

Additionally, the following options appear if the *Opportunities* option has been enabled in the

> *CRM* app's Configuration settings.

Reviewer 2: ```suggestion
**CRM** app's Configuration settings.
```

campaigns, salespeople, and more.

.. note::
  The **CRM** app supports using leads as an intermediate qualifying step before creating a formal
  opportunity. For the purposes of this article, the term "opportunity" is used, but the information
  applies to leads as well. This article assumes :ref:`leads have been enabled
  <crm/configure-leads>` in the **CRM** app's configuration settings.

.. _win_loss/pipeline:

Pipeline analysis reports
=========================

To view the *Pipeline Analysis* page, go to :menuselection:`CRM app --> Reporting --> Pipeline`. A
stacked bar chart showcasing all opportunities created during the current year automatically loads.
The bars represent the number of opportunities currently in each stage of the sales pipeline,
color-coded to show the month the opportunity reached that stage.

.. image:: win_loss/pipeline-analysis-page.png
   :alt: The default state of the Pipeline Analysis report is a graph, with many options to change
         it.

The interactive elements of the :guilabel:`Pipeline Analysis` page manipulate the graph to report
different metrics in different views. View options are in the upper-right corner.

> - :icon:`fa-area-chart` :guilabel:`Graph` view: Displays the total number of opportunities in a bar

Author: Thanks!

tell at a glance when leads and opportunities are lost or won and which salespeople were handling
them at the time.

To create the report, navigate to :menuselection:`CRM app --> Reporting --> Pipeline`. On the
:guilabel:`Pipeline Analysis` page, click the :icon:`fa-sort-down` :guilabel:`(Toggle Search Panel)`
icon at the end of the search bar to open a drop-down menu of filters and groupings. Under the
:icon:`fa-pencil` :guilabel:`Filters` heading, ensure both :guilabel:`Opportunities` and
:guilabel:`Leads` have been enabled to capture results for both Opportunities and Leads. Click
:guilabel:`Custom Filter...` to bring up the *Custom filter* pop-up. Click on the first field in the
:guilabel:`Match any of the following rules:` section and select :guilabel:`Won/Lost`. Doing so
automatically sets the second field to :guilabel:`is equal to`. Set the third field to
:guilabel:`Won` and add another rule by clicking :guilabel:`New Rule`. Configure this new rule the
same as the first one, except be sure to set the third field to :guilabel:`Lost`. Finally, toggle
the :guilabel:`Include archived` switch in the top-right of the form and click :guilabel:`Search`.

.. important::

>    Toggling the :guilabel:`Include archived` switch when creating win/loss reports guarantees that

Reviewer: ```suggestion
   Activating the :guilabel:`Include archived` toggle when creating win/loss reports guarantees that
```
Just because if it was active, clicking it woudl turn it off - so I just want ot clarify that.

   any archived *Lost* opportunities appear in the report. Without using the toggle or creating a
   rule that filters for both active and inactive opportunities, it's possible for a report to not
   include lost opportunities even with the :guilabel:`Lost` filter active.

In the search bar, choose :guilabel:`Stage` under the :icon:`oi-group` :guilabel:`Group By` heading,
activate the :guilabel:`Stage` option. Then click :guilabel:`Custom Group` and choose
:guilabel:`Won/Lost` from the drop-down menu. Finally, activate the :guilabel:`Salesperson` option.
By default, the data will appear with different results stacked on top of each other according to
their stage. To view the bars in this chart side-by-side, deactivate the :icon:`fa-database`
:guilabel:`(Stacked)` toggle.

> .. image:: win_loss/basic-win-loss-report.png

Reviewer: I recommend having the image at the end of the section for this example - so right above the next segment/header.

I als ot hink this could be in an example admonition (the image) and the text for this oculd be:

In this report, it can be detemrined that Marc Dmeo successfully closed the most leads (marked *Won*) while Michel Admin....

I am not 100% sure what the info shows isnce my eyesight is still.. not great - and I cna't read the itny tex.t BUT I cna see that the highest color bar is the best performer  =)
